packages/app/src/overview/components/update-os-modals/update-available-modal/about-updates-section.component.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import styled from "styled-components"
+import { defineMessages } from "react-intl"
+import { FunctionComponent } from "App/__deprecated__/renderer/types/function-component.interface"
+import Text, {
+  TextDisplayStyle,
+} from "App/__deprecated__/renderer/components/core/text/text.component"
+import { borderColor } from "App/__deprecated__/renderer/styles/theming/theme-getters"
+
+const messages = defineMessages({
+  updateAvailableAboutUpdatesTitle: {
+    id: "module.overview.updateAvailableAboutUpdatesTitle",
+  },
+  updateAvailableAboutUpdatesOsVersionDescription: {
+    id: "module.overview.updateAvailableAboutUpdatesOsVersionDescription",
+  },
+  updateAvailableAboutUpdatesTargetOsVersionSubDescription: {
+    id: "module.overview.updateAvailableAboutUpdatesTargetOsVersionSubDescription",
+  },
+})
+interface Release {
+  version: string
+  date: Date
+}
+
+export interface AboutUpdatesSectionProps {
+  releases: Release[]
+}
+
+const AboutUpdatesSectionTitle = styled(Text)`
+  border-bottom: 0.1rem solid ${borderColor("secondary")};
+  margin-bottom: 1rem;
+`
+
+const AboutUpdatesSectionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: 2.4rem 0;
+
+  p {
+    text-align: left;
+  }
+`
+
+const UpdatesOsVersionContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  justify-content: space-between;
+`
+
+export const AboutUpdatesSection: FunctionComponent<
+  AboutUpdatesSectionProps
+> = ({ releases }) => {
+  return (
+    <AboutUpdatesSectionContainer>
+      <AboutUpdatesSectionTitle
+        displayStyle={TextDisplayStyle.Label}
+        color="primary"
+        message={messages.updateAvailableAboutUpdatesTitle}
```

Comment:
Again, according to design, this might be in singular or plural form (`ABOUT UPDATE` vs `ABOUT UPDATES`

Reply:
https://github.com/mudita/mudita-center/pull/1049/commits/b1e89e36ec965f086ebdbf036cf35ef7965c500d